ztree-diff.el

```diff
@@ -95,6 +95,8 @@ By default paths starting with dot (like .git) are ignored")
 (defvar-local ztree-diff-wait-message nil
   "Message showing while constructing the diff tree.")
 
+(defvar ztree-diff--ediff-previous-window-configuration nil
```

Comment:
there is no conventions on private variables in ztree to have double dashes, please remove one